lib/notifier/email.js

```diff
@@ -0,0 +1,102 @@
+const _ = require('lodash/fp')
+const utils = require("./utils")
+
+const email = require('../email')
+
+const {
+  PING,
+  STALE,
+  LOW_CRYPTO_BALANCE,
+  HIGH_CRYPTO_BALANCE,
+  CASH_BOX_FULL,
+  LOW_CASH_OUT
+} = require('./codes')
+
+function alertSubject(alertRec, config) {
```

Comment:
Missing space before the argument list

```suggestion
function alertSubject (alertRec, config) {
```